addons/resource.language.en_gb/resources/strings.po

```diff
@@ -7712,7 +7712,19 @@ msgctxt "#13476"
 msgid "16 bit per channel"
 msgstr ""
 
-#empty strings from id 13477 to 13504
+#. Some Android devices have a bug and do not correctly report the capabilities (profile/level combinations it supports) of the codec component.
+#: system/settings/settings.xml
+msgctxt "#13477"
+msgid "Bypass the capabilities reported by the codec - MediaCodec"
```

Comment:
do you think that this setting could be reused for other use cases in the future? remove "MediaCodec" could be an idea
```suggestion
msgid "Bypass the capabilities reported by the codec"
```

Reply:
I think it's positive to keep it as it only affects decoding using the Android Media Codec API.

Other related settings also include:

https://github.com/xbmc/xbmc/blob/10a908e7498369172ced4723818e3816cc3eefba/addons/resource.language.en_gb/resources/strings.po#L7515

https://github.com/xbmc/xbmc/blob/10a908e7498369172ced4723818e3816cc3eefba/addons/resource.language.en_gb/resources/strings.po#L7520